example/src/App.tsx

```diff
@@ -7,7 +7,8 @@ import {
   ScrollView,
   SafeAreaView,
 } from 'react-native';
-import { ClickstreamAnalytics, Item } from 'clickstream-react-native';
+// @ts-ignore
+import { ClickstreamAnalytics, Item } from '@aws/clickstream-react-native';
```

Comment:
Let's add a workflow to build example code for every PR.

Reply:
changed the root project `tsconfig.json` file, add `example` as composite project. The new `build-rn` action will build example project.